packages/logout-extension/src/index.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  JupyterFrontEnd,
+  JupyterFrontEndPlugin,
+  IRouter,
+} from '@jupyterlab/application';
+
+import '@jupyterlab/application/style/buttons.css';
+
+import '../style/index.css';
+
+const logoutPluginId = '@jupyterlab/logout-extension:plugin';
+
+const extension: JupyterFrontEndPlugin<void> = {
+  id: logoutPluginId,
+  autoStart: true,
+  requires: [IRouter],
+  activate: async (app: JupyterFrontEnd, router: IRouter): Promise<void> => {
+    console.log('@jupyterlab/logout-extension is activated!');
+
+    // Get app commands
+    const { commands } = app;
+
+    const namespace = 'topbarcmds';
```

Comment:
For consistency with other third-party extensions, should we rename the namespace to `jupyterlab-logout`?

```suggestion
    const namespace = 'jupyterlab-logout';
```

Reply:
What do you think about `jupyterlab-topbar` as namespace? It will still be project specific namespace and we will be able to use same namespace for any future buttons we will add to topbar.

Reply:
That could indeed work, unless we consider all these extensions as independent and the first part of the command (namespace) to correspond to the name of the extension.